### PR TITLE
Guide to create custom models in Spanish

### PR DESCRIPTION
**Pull request by ignacioct**
# What does this PR do?

Linked to #15947

## Before submitting
- [X ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

## Who can review?

@omarespejel said he will review, otherwise anyone 😁.

### CODE REVIEW

File: docs/source/es/create_a_model.mdx [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+    0.5
+  ],
+  "resample": 2,
+  "size": 224
+}
+```
+
+<Tip>
+
+Si no estas buscando ninguna personalización en específico, usa el método `from_pretrained` para cargar los parámetros del extractor de características por defecto del modelo.
```
Comment (omarespejel) [inline docs/source/es/create_a_model.mdx:289]:
```suggestion
Si no estás buscando ninguna personalización en específico, usa el método `from_pretrained` para cargar los parámetros del extractor de características por defecto del modelo.
```

File: docs/source/es/create_a_model.mdx [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+
+Puedes combinar el extractor de características y el *tokenizer* en el [`Wav2Vec2Processor`]:
+
+
+```py
+>>> from transformers import Wav2Vec2Processor
+
+>>> processor = Wav2Vec2Processor(feature_extractor=feature_extractor, tokenizer=tokenizer)
+```
+Con dos clases base (la configuración y el modelo) y una clase de preprocesamiento adicional (*tokenizer*, extractor de características o procesador), puedes crear cualquier de los modelos compatibles con 🤗 Transformers. Cada una de estas clases son configurares, permitiéndote usar sus atributos específicos. Puedes crear un modelo para entrenarlo de una forma fácil, o modificar un modelo preentrenado disponible para especializarlo.
```
Comment (omarespejel) [inline docs/source/es/create_a_model.mdx:367]:
```suggestion
Con dos clases base (la configuración y el modelo) y una clase de preprocesamiento adicional (*tokenizer*, extractor de características o procesador), puedes crear cualquiera de los modelos compatibles con 🤗 Transformers. Cada una de estas clases son configurares, permitiéndote usar sus atributos específicos. Puedes crear un modelo para entrenarlo de una forma fácil, o modificar un modelo preentrenado disponible para especializarlo.
```

**Review by omarespejel — CHANGES_REQUESTED** [review]

**Comment by ignacioct** [comment]
Thanks for the suggestions, I've committed all the changes you proposed.

**Comment by omarespejel** [comment]
@ignacioct muchas gracias por tu PR! 🤗 Vamos a hacerle merge. If there is any other doc that you would like to translate you can tell me. 

@sgugger LGTM 🔥

**Review by sgugger — APPROVED** [review]

**Pull request merged**